prow/cmd/horologium/main.go

```diff
@@ -170,7 +170,13 @@ func sync(prowJobClient ctrlruntimeclient.Client, cfg *config.Config, cr cronCli
 		})
 
 		if p.Cron == "" {
-			shouldTrigger := j.Complete() && now.Sub(j.Status.StartTime.Time) > p.GetInterval()
+			intervalRef := j.Status.StartTime.Time
```

Comment:
this looks good to me, would need unit test for this as well, ideally we could even add integration test that is similar to https://github.com/kubernetes/test-infra/blob/62ebc9c2bee6f61df2d96b0dc7ca701a7badab88/prow/test/integration/test/horologium_test.go#L50

Reply:
Good call for adding unit tests, thanks to them I spotted and fixed an issue with the implementation ;-)
I believe unit tests provide enough coverage here, but could look into adding an integration test as well if you insist.